### DISCUSSION TITLE
Hovercards stop disappearing after navigating to an issue page

### DISCUSSION THREAD

**Discussion by bojidar-bg** [community/community#44561] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I'm... honestly not sure if this should go here. (And yep, that's a bit of feedback too, please make it more obvious where we are supposed to direct UI/UX bugs to :pray:)

But, I've been noticing that hovercards (the boxes that appear when hovering over a username/mention) sometimes don't hide when one moves their mouse away. Not the end of the world, but those would occasionally hide important things on the page, and not being able to hide them gets in the way. The issue is super intermittent, and I've failed to find a reliable way to reproduce it so far -- it just seems to happen on 1-10% of page loads. It always seems to go away on refreshing the page (or maybe it's just that rare), and it sometimes seems to sort itself out when selecting/clicking things on the page.

Here's a video of me hovering around after randomly seeing the bug on a page: [Peek 2023-01-17 11-11.webm](https://user-images.githubusercontent.com/5276727/212857968-1c4b6289-2978-477f-b478-60b4fde9e565.webm) (taken on [this issue](https://github.com/pulseaudio-equalizer-ladspa/equalizer/issues/28)) -- note how halfway through the video, the bug randomly fixes itself right after I select some text, but right before that, I do a similar selection with no result. ~~Also, might be of note, but other users' nicknames on the video don't trigger any hovercards, though I'm not sure if that's related or a separate thing.~~ Update: that's unrelated.

For what's worth, I currently have Colorblind Themes and Command Pallette feature previews on, and I'm on a dark theme; using the uBlock Origin and KeePass extenstions on Firefox 108.0.2 (64-bit) on Arch Linux.

EDIT (2023-01-24): I just managed to get another reproduction; the problem went away after clicking on/selecting text in the hovercard itself. Not sure how that's related to my video earlier tho; either it's a bug with multiple possible causes, or the fact that my first selection on the video ended with my mouse on top of the hovercard somehow fixed the bug halfway and the second selection on the text outside the hovercard got it to hide.

EDIT (2023-01-30): After discussing with other users in the replies below, I can finally reproduce this at a 100% rate. Just open an issue (e.g. [this one](https://github.com/embee8/homebridge-panasonic-ac-platform/issues/27), though others seem to work too) from a link, in the same tab. I'm guessing something is off with how GitHub manages history, since the issue doesn't occur (for me) when opening the issue in a new tab or refreshing a broken tab.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bojidar-bg ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

Also - appreciate the feedback, for now UI/UX feedback, bugs, etc can go in the General category unless it specific to a feature area of our Product categories. Keep the candidness coming 😄

**Reply by bojidar-bg** [reply]
Ah, thanks for moving it to the right place! Definitely wouldn't have guessed it should go in General :sweat_smile: Hopefully, the bug will get resolved at some point; but till then, keep up the good work! (:

**Comment by embee8** [comment]
I would simply have upvoted this topic, but this bug is so detrimental to my user experience and productivity that I will also echo my frustration in a comment. It would be great if the team in charge of the [hovercards](https://blog.github.com/2018-03-22-user-hovercards/) could look into this.

**Reply by bojidar-bg** [reply]
Hm, I only rarely run into it; are you perchance managing to reproduce it more often than a ~tenth of page loads, or is your work process simply different than mine? Just making sure we aren't missing out having exact reproduction steps here. (:

**Reply by embee8** [reply]
Yes, I can reproduce it consistently after every fresh page load of an Issue [in one of my repos](https://github.com/embee8/homebridge-panasonic-ac-platform/issues/27).

Interestingly, I cannot reproduce this behaviour in the Issue we are currently discussing. When I hover over a name, I see the card; when I move the cursor away, it disappears within 50-200 ms. Not sure if slightly different front-end components are used for different repositories or whether it's a cached bug. I tried force-refreshing the page above but still run into the same issue.

Edit: When I navigate from this issue (where the hovercards work as intended) to the issue linked above (where they don't work) and navigate back to this issue, the hovercards are also broken here.

**Reply by brieucdlf** [reply]
Hey!

I constantly reproduce this on Firefox v108.x but not on chromium browser if it can help ;)

**Reply by bojidar-bg** [reply]
Wow, there is something with that! :sparkles: If I go to it through the link you posted or through the issue list on your repo, it reproduces always. However, if I open it in a new tab (or similarly, if I hit F5), it doesn't reproduce. (Testing on other issues, it seems to work similarly there too. Hm!)
Edit: In fact, hovercards in the issues list seem to be similarly broken now. Odd. I thought I tested simple stuff like that before opening the discussion, but either I missed it or it's gotten worse since. XD

**Comment by bojidar-bg** [comment, marked as answer]
Aactualy, this seems to be a duplicate of #44630 and is being worked on as per https://github.com/orgs/community/discussions/44630#discussioncomment-4729878. Going to close it as such. :sweat_smile: 

(For an extra note of feedback: Using Conversations for issues, like here, is rather clunky. There is a reason the Close and Comment button, for example, is super useful with issues, just like having a single comment thread is great for collaborating on reproduction steps and so on. Just saying (: )
